include/fiction/algorithms/simulation/sidb/random_sidb_layout_generator.hpp

```diff
@@ -83,18 +83,18 @@ struct generate_random_sidb_layout_params
 };
 
 /**
- * Generates a random layout of SiDBs by adding them to the provided layout skeleton.
- * The layout skeleton serves as the starting layout to which SiDBs are added to create the final layout.
+ * Generates a random layout of SiDBs by adding them to the provided layout layout.
```

Comment:
How about: "Generates a layout featuring a random arrangement of SiDBs. These randomly placed dots can be incorporated into an existing layout skeleton that may be optionally provided."?

As mentioned before, I'd stay with the `skeleton` naming as `layout` is less self-explanatory to me.